### PR TITLE
ENH: tseries with shutter control

### PR DESCRIPTION
**Pull request by chiahaoliu**
This PR implements shutter control in ``tseries`` scan (time series). It provides default behavior of opening shutter before triggering the detectors and closing shutter after saving data to centralized database at each event (count).

### CODE REVIEW

**Comment by chiahaoliu** [comment]
@sbillinge @CJ-Wright Can I have a review on this PR? Thanks.

File: xpdacq/beamtime.py [hunk 1]
```diff
@@ -371,6 +403,18 @@ def tseries(dets, exposure, delay, num):
                         'sp_plan_name': 'tseries'})
     plan = bp.count([area_det], num, delay, md=_md)
     plan = bpp.subs_wrapper(plan, LiveTable([]))
+    def inner_shutter_control(msg):
+        if msg.command=='trigger':
```
Comment (CJ-Wright) [inline xpdacq/beamtime.py]: I think this needs spaces around the equality.

Comment (chiahaoliu) [reply xpdacq/beamtime.py]: yeah, thanks!

File: xpdacq/beamtime.py [hunk 2]
```diff
@@ -404,12 +404,12 @@ def tseries(dets, exposure, delay, num, auto_shutter=True):
...
     plan = bpp.subs_wrapper(plan, LiveTable([]))
     def inner_shutter_control(msg):
-        if msg.command=='trigger':
+        if msg.command =='trigger':
             def inner():
                 yield from _open_shutter_stub()
                 yield msg
             return inner(), None
-        elif msg.command=='save':
+        elif msg.command =='save':
```
Comment (CJ-Wright) [inline xpdacq/beamtime.py]: I think this needs a space on the either side too.

**Pull request merged**